clientv3/integration/kv_test.go

```diff
@@ -895,3 +895,31 @@ func TestKVGetResetLoneEndpoint(t *testing.T) {
 	case <-donec:
 	}
 }
+
+// TestKVPutLargeRequests ensures that custom MaxRequestBytes works as intended.
+func TestKVPutLargeRequests(t *testing.T) {
```

Comment:
this probably belongs in github.com/coreos/etcd/integration since it's only testing server functionality